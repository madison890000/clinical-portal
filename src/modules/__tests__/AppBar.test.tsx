import React from 'react';
import AppBar from '../AppBar';
import { render, waitFor } from '@testing-library/react';
import wrapper from '../../tests/reactQueryWrapper';
import { BrowserRouter } from 'react-router-dom';

test('renders PatientDetail correctly', async () => {
    const { container } = render(
        <BrowserRouter>
            <AppBar />
        </BrowserRouter>,
        { wrapper }
    );
    await waitFor(() => {
        expect(container).toMatchSnapshot();
    });
});
