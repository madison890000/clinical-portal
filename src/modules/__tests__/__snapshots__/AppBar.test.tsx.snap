// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PatientDetail correctly 1`] = `
<div>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthXl css-19r6kue-MuiContainer-root"
    >
      <div
        class="MuiToolbar-root MuiToolbar-regular css-r6ewbb-MuiToolbar-root"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1uzrjc8-MuiSvgIcon-root"
          data-testid="AdbIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M5 16c0 3.87 3.13 7 7 7s7-3.13 7-7v-4H5v4zM16.12 4.37l2.1-2.1-.82-.83-2.3 2.31C14.16 3.28 13.12 3 12 3s-2.16.28-3.09.75L6.6 1.44l-.82.83 2.1 2.1C6.14 5.64 5 7.68 5 10v1h14v-1c0-2.32-1.14-4.36-2.88-5.63zM9 9c-.55 0-1-.45-1-1s.45-1 1-1 1 .45 1 1-.45 1-1 1zm6 0c-.55 0-1-.45-1-1s.45-1 1-1 1 .45 1 1-.45 1-1 1z"
          />
        </svg>
        <a
          class="MuiTypography-root MuiTypography-h6 MuiTypography-noWrap css-m5x0z-MuiTypography-root"
          href="/"
        >
          Clinical Portal
        </a>
        <div
          class="MuiBox-root css-1lvtzne"
        >
          <button
            aria-controls="menu-appbar"
            aria-haspopup="true"
            aria-label="account of current user"
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-sizeLarge css-1pz7awu-MuiButtonBase-root-MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="MenuIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
              />
            </svg>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-z5gn4u-MuiSvgIcon-root"
          data-testid="AdbIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M5 16c0 3.87 3.13 7 7 7s7-3.13 7-7v-4H5v4zM16.12 4.37l2.1-2.1-.82-.83-2.3 2.31C14.16 3.28 13.12 3 12 3s-2.16.28-3.09.75L6.6 1.44l-.82.83 2.1 2.1C6.14 5.64 5 7.68 5 10v1h14v-1c0-2.32-1.14-4.36-2.88-5.63zM9 9c-.55 0-1-.45-1-1s.45-1 1-1 1 .45 1 1-.45 1-1 1zm6 0c-.55 0-1-.45-1-1s.45-1 1-1 1 .45 1 1-.45 1-1 1z"
          />
        </svg>
        <a
          class="MuiTypography-root MuiTypography-h5 MuiTypography-noWrap css-1lfs2tq-MuiTypography-root"
          href=""
        >
          Clinical Portal
        </a>
        <div
          class="MuiBox-root css-1t6c9ts"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1q39md6-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            Menu1
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1q39md6-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            Menu2
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
        <div
          class="MuiBox-root css-2uchni"
        >
          <button
            aria-label="Open settings"
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-av538e-MuiButtonBase-root-MuiIconButton-root"
            data-mui-internal-clone-element="true"
            tabindex="0"
            type="button"
          >
            <div
              class="MuiAvatar-root MuiAvatar-circular css-1wlk0hk-MuiAvatar-root"
            >
              <img
                alt="Remy Sharp"
                class="MuiAvatar-img css-1pqm26d-MuiAvatar-img"
                src="/static/images/avatar/2.jpg"
              />
            </div>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
    </div>
  </header>
</div>
`;
